jenkins/bootstrap.py

```diff
@@ -735,11 +737,16 @@ def job_args(args):
     return [os.path.expandvars(a) for a in args]
 
 
-def job_script(job, use_json):
+def job_script(job, use_json, jobs_dir):
     """Return path to script for job."""
+    jobs_dir_resolver = lambda f: test_infra('jobs/%s' % f)
+
+    if jobs_dir:
+        jobs_dir_resolver = lambda f: os.path.join(jobs_dir, f)
+
     if not use_json:
```

Comment:
I don't follow...it shouldn't be an error, it's the core use case.

use_json is false means that we are looking for a shell script to run...One aim of this PR is to allow that shell script to live in the individual repository under test and not in the k8s/test-infra repo...

What am I missing?